Templates/Kotlin/GraphApi.stencil

```diff
@@ -15,15 +15,25 @@ import java.lang.reflect.Type
 
 const val DEFINED_NULL_FLAG = "__DEFINED_NULL"
 
-interface SyrupOperation<T: Response> {
-    fun getQueryString(): String
+interface SyrupOperation<T : Response> {
+    val rawQueryString: String
+    fun getQueryString(): String {
```

Comment:
Suggest moving to all properties here too.

Reply:
to clarify, make getQueryString a computed property?

Reply:
going to make this change too
https://github.com/Shopify/android/pull/14076/files#r419182367